opengrok-indexer/src/main/jflex/analysis/fortran/FortranXref.lex

```diff
@@ -95,6 +95,14 @@ File = [a-zA-Z]{FNameChar}* ".inc"
     onNonSymbolMatched(">", yychar + 1 + file.length());
 }
 
+"'" ({File}|{FPath}) "'" {
```

Comment:
This should be combined with the rule (by using a regexp for the leading/terminating characters) above to avoid code duplication.

Reply:
If I say. <|' then how to capture what matched so that I can use it below like \1 .Any reference ?

Reply:
Just grab it from the string returned by `yytext()`. It should contain the whole matched input.